### PR TITLE
Update configure-teams-live-events.md

### PR DESCRIPTION
**Pull request by chbalaki**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-live-events/configure-teams-live-events.md [hunk 1]
```diff
@@ -91,7 +91,10 @@ For more information, see [Set-CsTeamsMeetingBroadcastConfiguration](/powershell
 >[!Note]
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/configure-teams-live-events.md:91]:
```suggestion
> [!NOTE]
```

File: Teams/teams-live-events/configure-teams-live-events.md [hunk 2]
```diff
@@ -91,7 +91,10 @@ For more information, see [Set-CsTeamsMeetingBroadcastConfiguration](/powershell
...
 > The change from using Microsoft Stream to [OneDrive for Business and SharePoint for meeting recordings](../tmr-meeting-recording-change.md) will be a phased approach. At launch you'll be able to opt-in to this experience, in November you'll have to opt-out if you want to continue using Stream, and some time in early 2021 we'll require all customers to use OneDrive for Business and SharePoint for new meeting recordings.
 
+>[!Note]
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/configure-teams-live-events.md:94]:
```suggestion
> [!NOTE]
```

File: Teams/teams-live-events/configure-teams-live-events.md [hunk 3]
```diff
@@ -91,7 +91,10 @@ For more information, see [Set-CsTeamsMeetingBroadcastConfiguration](/powershell
...
+> Your chosen eCDN solution is subject to the selected 3rd party provider’s terms of service and privacy policy, which will govern your use of the eCDN provider’s solution. Your use of the eCDN provider’s solution will not be subject to the Microsoft volume licensing terms or Online Services Terms. If you don't agree to the 3rd party provider’s terms, then don't enable the eCDN solution in Microsoft Teams.
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/configure-teams-live-events.md:95]:
```suggestion
> Your chosen eCDN solution is subject to the selected third-party provider’s terms of service and privacy policy, which will govern your use of the eCDN provider’s solution. Your use of the eCDN provider’s solution will not be subject to the Microsoft volume licensing terms or Online Services Terms. If you don't agree to the third-party provider’s terms, then don't enable the eCDN solution in Microsoft Teams.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chbalaki please also commit my minor additional suggested changes. Thanks.

**Pull request merged**